packages/trip-form/src/SettingsSelectorPanel/index.js

```diff
@@ -82,7 +82,8 @@ export default class SettingsSelectorPanel extends Component {
       }
 
       const nonTransitModes = newModes.length > 1 ? [newModes[1]] : ["WALK"]; // TODO: also accommodate WALK+DRIVE, WALK+e-scooter?? They already seem to work without WALK right now.
-      const defaultCompany = newModes.length > 2 ? [newModes[2]] : null; // To accommodate companies defined under accessModes.
+      const defaultAccessModeCompany =
+        newModes.length > 2 ? [newModes[2]] : null; // To accommodate companies defined under accessModes.
```

Comment:
This is just a minor style thing, so not necessary to change, but I generally prefer comments to be above the line not next to the line. Just improves legibility in a code editor.